build/changelog/build-changelog.php

```diff
@@ -4,8 +4,14 @@
 
 use Aws\Build\Changelog\ChangelogBuilder;
 
-$changelogBuilder = new ChangelogBuilder("", "", true);
+$params = [
+    'verbose_flag' => true
+];
+
+$changelogBuilder = new ChangelogBuilder($params);
```

Comment:
Is adding a todo any less work than adding a call to `get_opt`? At the very least, you should accept a `-v` flag instead of always setting `verbose` to `true`. Additionally, if no option to configure inputs and outputs is offered, the todo comment should specifically call out from which directory this script expects to be run.